frontend/src/components/LinkModule.tsx

```diff
@@ -45,22 +46,37 @@ export default function LinkModule(props: LinkModuleProps) {
     <div className="link-card">
       <img
         src={committeeLogos[props.committee as keyof CommitteeDict]}
```

Comment:
I was going to make this a separate issue because getting the correct logo is actually fairly nuanced now that we're getting the committee from Github API